tests/cli/run/ensure-absolute-inputs/outputs/output/inputs.json

```diff
@@ -0,0 +1,4 @@
+{
+  "first": "/private/var/folders/9t/yj3jt2pn5zx5mzpy6hqc8f540000gp/T/.tmpUqRWPX/files/first.txt",
```

Comment:
Is this right?

Reply:
To be clear, my question is regarding the path itself. It seems strange to commit such a specific path and that be intentional/the right thing.

Reply:
Sigh, it definitely is not. The current run tests don't do much, so they didn't need to normalize the output files much.

Let me figure out the best way to get these tests running in a way that works across the various platforms.